docs/installation.rst

```diff
@@ -83,7 +92,7 @@ seemly successful installation with apexpy reporting that fortranapex cannot be
 imported.
 
 ::
-   LDFLAGS="-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib ${LDFLAGS}" /opt/local/bin/python3.10 setup.py install
+   LDFLAGS="-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib ${LDFLAGS}" python setup.py install
```

Comment:
Should we stop recommending using setup.py in the installation docs?
```suggestion
   LDFLAGS="-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib ${LDFLAGS}" pip .
```

Reply:
Probably, I wasn't sure about that one... A carry-over copy-paste?

Reply:
Given the other flags I've used with pip, I think it will work.

Reply:
Or maybe it would go with the build command (more likely).

Reply:
I'm not sure, I don't think I ever encountered this issue...

Reply:
Let's leave it alone then, since I am not certain 😨

Reply:
Ugg, I don't particularly want to leave in a recommendation for people to use setup.py - everything I've read seems to indicate that's a bad idea now.  Can we use your original suggestion but move it down to the GitHub section?  Probably should move it regardless.  Alternatively we could create an archive of "installation commands people have reported working but we're not entirely sure why"?

Reply:
I deeply feel that: "installation commands people have reported working but we're not entirely sure why" is appropriate.